src/app/tests/suites/commands/interaction_model/InteractionModel.h

```diff
@@ -264,14 +288,15 @@ class InteractionModelWriter
     CHIP_ERROR WriteAttribute(chip::DeviceProxy * device, std::vector<chip::EndpointId> endpointIds,
                               std::vector<chip::ClusterId> clusterIds, std::vector<chip::AttributeId> attributeIds, const T & value,
                               const chip::Optional<uint16_t> & timedInteractionTimeoutMs          = chip::NullOptional,
+                              const chip::Optional<uint16_t> & busyWaitForMs                      = chip::NullOptional,
```

Comment:
and this one now has 11 arguments. ... same complaint: this is not long term maintanable and invocations of this API is not code reviewable for correctness.

Reply:
@andy31415  I made https://github.com/project-chip/connectedhomeip/pull/24091 to address this issue. I will rebased this one on ToT once https://github.com/project-chip/connectedhomeip/pull/24091 has landed.

Reply:
I have rebased this patch on top of #24091 now that it has landed.